languages/go/exercises/concept/conditionals/.docs/introduction.md

```diff
@@ -9,6 +9,3 @@ if condition {
 ```
 
 In scenarios involving more than one case many `if` statements can be chained together using the `else if` and `else` statements. Go also provides a `switch` statement for more advanced scenarios.
```

Comment:
The `else if` and `switch` statement are new syntax, which means that there should be examples to show their syntax in this document.